index.js

```diff
@@ -160,6 +158,7 @@ class GoldenSun {
 
         //initialize screens
         this.menu_screen = initialize_menu(this.game, this);
+        this.field_psynergy_window = new FieldPsynergyWindow(this.game, this);
```

Comment:
I'm avoiding creating variables inside GoldenSun class, the past month I've been reducing a lot these variables. This a variable that I would set in another place. 

In my plans for refactoring field abilities, I was planning to do a parent class FieldAbilities where the specifics field abilities would inherit from it. As you can see, there are many things that are repeating among these classes. This new class would be a suitable place for `field_psynergy_window `.